test/fixtures/output/iam/group-mapping/create.golden

```diff
@@ -0,0 +1,9 @@
++-------------+--------------------------------+
+| ID          | pool-12345                     |
+| Name        | group_mapping                  |
+| Description | new-group-description          |
```

Comment:
We have a ticket to include this output in our public-facing documentation, so let's make sure the formatting matches the style of our examples.
```suggestion
| Name        | my-group-mapping               |
| Description | new-description                |
```

Reply:
Can you link the ticket? Also IIUC, you mean our CRUDL integration test output should match what is in our examples?

Reply:
I mean the examples and integration tests should all be formatted with lower-kebab-case and have similar fake resource names / IDs.

Reply:
oh ok gotcha